src/ripple/core/DatabaseCon.h

```diff
@@ -135,15 +151,28 @@ class DatabaseCon
     setupCheckpointing(JobQueue*, Logs&);
 
 private:
+    class Base
+    {
+    };
+
     template <std::size_t N, std::size_t M>
-    void
-    init(
+    DatabaseCon(
+        Base,
```

Comment:
I don't understand the motivation for adding the `Base` parameter. The function doesn't appear to be ambiguous without it. Unless I'm missing the reason, can we remove this parameter?

Reply:
Basically, if I continue to use `{}` to specify the `commonPragma` param, then it is ambiguous. However, because of the rewrite in the last commit to use a raw pointer, it is not ambiguous if I use `nullptr`, so I've removed `Base`, and changed the caller to use `nullptr`.